web/src/lib/utils/file-uploader.ts

```diff
@@ -40,11 +40,11 @@ export const openFileUploadDialog = (uploadType: UploadType) => {
 			}
 			const files = Array.from<File>(target.files);
 
-			if (files.length > 50) {
+			if (files.length > 200) {
```

Comment:
Would this too risky for low memory machine? :P It could crash the broswer.

Reply:
I'm confident it can reach to this limit with my new algorithm.